src/docs/truffle/getting-started/running-migrations.md

```diff
@@ -187,16 +193,16 @@ Examples:
 
 ```javascript
 // Deploy a single contract without constructor arguments
-deployer.deploy(A);
+async deployer.deploy(A);
```

Comment:
All of these `async`s should be `await`s. This includes on line 199, 202, 205, 234, and 236.